actions/project.go

```diff
@@ -69,7 +86,56 @@ func DownloadTemplate(c *cli.Context) {
 	utils.DeleteTempFile(zipFileName)
 }
 
-//ValidateProject type
-func ValidateProject() {
-	//code here
+// ValidateProject returns the language and buildType for a project at given filesystem path
+func ValidateProject(c *cli.Context) {
+	projectPath := c.Args().Get(0)
+	language, buildType := determineProjectInfo(projectPath)
+
+	resp := ValidationResponse{
+		Status: "success",
+		Result: ProjectType{language, buildType},
+		Path:   projectPath,
+	}
+
+	projectInfo, err := json.Marshal(resp)
+	if err != nil {
+		log.Fatal(err)
+		return
+	}
+	fmt.Println(string(projectInfo))
+}
+
+func determineProjectInfo(projectPath string) (string, string) {
+	language := "unknown"
+	buildType := "docker"
+	if _, err := os.Stat(path.Join(projectPath, "pom.xml")); err == nil {
```

Comment:
Could this possibly be a switch statement? - might look neater and easier to maintain

Reply:
What would you switch on?